core/src/main/scala/org/ensime/indexer/stringymap/StringyMap.scala

```diff
@@ -68,6 +78,7 @@ package object impl {
     def label: String = ???
     def toProperties(t: HNil): StringyMap = new java.util.HashMap()
     def fromProperties(m: StringyMap) = Right(HNil)
+    def toSchema: Map[String, (OType, Boolean)] = Map()
```

Comment:
`Map.empty` is more efficient. For fun, follow through the code that gets invoked when you call `Map()`.